Sources/Passage/AuthorizationController/PassageSocialAuthController.swift

```diff
@@ -70,6 +70,10 @@ final internal class PassageSocialAuthController:
     
     // MARK: STATIC METHODS
     
+    internal static func getCallbackUrlScheme(appId: String) -> String {
+        return "passage-\(appId)"
```

Comment:
Just checking - this doesn't need a closing slash or anything? I forget how this works in swift!

Reply:
Nope, no closing slash needed :)